### DISCUSSION TITLE
Depremzedelere evini açmak isteyenler için platform.

### DISCUSSION THREAD

**Discussion by dev-fatih-erol** [acikkaynak/deprem-yardim-projesi#71] [Ideas]
Depremzedelere başka şehirlere taşımak isteyen, Onlara evini açmak isteyen binlerce yardımsever var. 
Tahliye olmak isteyenlerle bu yardımseverleri buluşturacak bilinir bir platform oluşturula bilir.

**Comment by sercangoger** [comment]
https://deprem.io/
yardım listesi bölümünde bir çoğu mevcut incelermisiniz. ona göre eksik olanı acıklayıp arkadaşlara sunabilirsiniz.
teşekkürler
